### ISSUE TITLE
Multi selection of nodes.

### ISSUE THREAD

**Issue by jitenderd** [valor-software/ng2-tree#207]
I want to implement multi selection of nodes using ctrl+left mouse click. Currently we are not having this feature with ng2-tree. Is there any possible way to do this. if yes please suggest.

**Comment by rychkog** [comment]
@jitenderd There is a PR from @TMaster. Which potentially might be what you are seeking for

**Comment by gomedh** [comment]
Yes, even I need this feature to be implemented. Please notify once the work is done.

**Comment by rychkog** [comment]
@gomedh Deal
